android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/RepeatingGroupFactory.java

```diff
@@ -78,7 +80,7 @@
 
     private ImageButton doneButton;
     private WidgetArgs widgetArgs;
-    private Map<String, JSONArray> readFileMap = new HashMap<>();
+    private Map<String, List<Map<String, Object>>> readFileListMap = new HashMap<>();
```

Comment:
Any reason this implementation changed?

IMO this `Map<String, List<Map<String, Object>>>` is much harder to read and understand than `Map<String, JSONArray>`.

Reply:
Here what am trying to achieve is to map a file name to processed contents of the yaml file, so here a string will be the file name and the List of maps will be the content of the yaml file .i.e map will have contents of one rule and the list will hold maps of several rules

Reply:
Ok, please use a more descriptive name for the `Collection` then.